yoga/Yoga.cpp

```diff
@@ -1695,6 +1695,15 @@ static void YGNodeWithMeasureFuncSetMeasuredDimensions(
       : YGFloatMax(
             0, availableHeight - marginAxisColumn - paddingAndBorderAxisColumn);
 
+#ifdef YOGA_CALL_MEASURE_CALLBACK_ON_ALL_NODES
+  const YGSize measuredSize = node->measure(
+      innerWidth,
+      widthMeasureMode,
+      innerHeight,
+      heightMeasureMode,
+      layoutContext);
+#endif
+
   if (widthMeasureMode == YGMeasureModeExactly &&
```

Comment:
This is still getting executed even if measure callback is called on all the nodes. 
I think you want to use measuredSize after measure callback